app/styles/ui/_branches.scss

```diff
@@ -4,14 +4,15 @@
   height: 100%;
   display: flex;
   flex-direction: column;
-  width: 365px;
+  min-width: 365px;
+  width: 100%;
```

Comment:
Replace the fixed width with minimum width so the branch toolbar dropdown always has at least 365px. If the toolbar button is wider than that, allow it to fill the same width.